pkg/workloads/qsfs.go

```diff
@@ -171,11 +185,11 @@ func NewQSFSFromSchema(qsfs map[string]interface{}) QSFS {
 		Name:                 qsfs["name"].(string),
 		Description:          qsfs["description"].(string),
 		Cache:                qsfs["cache"].(int),
-		MinimalShards:        uint32(qsfs["minimal_shards"].(int)),
-		ExpectedShards:       uint32(qsfs["expected_shards"].(int)),
-		RedundantGroups:      uint32(qsfs["redundant_groups"].(int)),
-		RedundantNodes:       uint32(qsfs["redundant_nodes"].(int)),
-		MaxZDBDataDirSize:    uint32(qsfs["max_zdb_data_dir_size"].(int)),
+		MinimalShards:        qsfs["minimal_shards"].(uint32),
```

Comment:
i had a discussion with @rawdaGastan about this. I guess the only types that terraform acknowledge are the ones mentioned here: <https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/valuetype.go>
so, casting to uint32 panics since the returned type is int.
maybe we could change the types in the QSFS struct itself??